llvm/lib/MC/MCObjectFileInfo.cpp

```diff
@@ -928,10 +928,16 @@ void MCObjectFileInfo::initXCOFFMCObjectFileInfo(const Triple &T) {
   // the ABI or object file format, but various tools rely on the section
   // name being empty (considering named symbols to be "user symbol names").
   TextSection = Ctx->getXCOFFSection(
-      "", SectionKind::getText(),
+      "..text..", // Use a non-null name to work around an AIX assembler bug...
+      SectionKind::getText(),
       XCOFF::CsectProperties(XCOFF::StorageMappingClass::XMC_PR, XCOFF::XTY_SD),
       /* MultiSymbolsAllowed*/ true);
 
+  // ... but use a null name when generating the symbol table.
```

Comment:
delete `...` here.  how about change the comment to 
`Using a null name instead of '..text..' when generating the symbol table.` ?

Reply:
The elipsis indicates a continuation of the previous comment:
// Use a non-null name to work around an AIX assembler bug...
// ... but use a null name when generating the symbol table.

Reply:
thanks for explain.  is there a refence url or standard about `The elipsis indicates a continuation of the previous comment:`?